src/bin.js

```diff
@@ -30,7 +30,7 @@ if (program.args.length !== 1) program.help();
 var token = program.args[0];
 
 var abort = function(err) {
-  console.error(err.message || err);
+  console.error(err);
```

Comment:
Could you hide this behind a `--debug` or other similarly named flag? As much as we benefit from the traceback, it is my believe that for regular users who are not looking for it, a traceback is needlessly loud.

If you haven’t used `commander` before, we need to add an `.option` call after line 14 and check it here like `console.error(program.debug ? err : err.message || err)`.

Reply:
I like how easy that is!